src/lib/bootloader/grub2_widgets.rb

```diff
@@ -787,6 +816,10 @@ def secure_boot_widget?
       (Yast::Arch.x86_64 || Yast::Arch.i386) && grub2.name == "grub2-efi"
     end
 
+    def trusted_boot_widget?
+      (Yast::Arch.x86_64 || Yast::Arch.i386) && grub2.name != "grub2-efi"
```

Comment:
I prefer to use `grub2.name != "grub2"` otherwise here can be confusion that it is shown by `"none"` bootloader